### DISCUSSION TITLE
Tanya-tanya

### DISCUSSION THREAD

**Discussion by el-ang** [lintangtimur/ovoid#52] [Q&A] [answered]
Halo! Salam dari Bali!

Saya mau nanya mengenai repo ini..
1. Apakah ini semacam library untuk Payment Gateaway OVO atau full API Aplikasi OVO?
2. Apakah ada function sistem pembayaran sesama akun atau ke merchant/outlet?
3. Informasi apa saja yang umumnya di assign untuk ID OVO nya?(Terutama pada transaksi)
4. Curl disini untuk naruh respon lewat function/act di folder Response ya?
5. Bagaimana cara menggunakan library/API ini? Apakah dengan assign classnya pada object? Atau ada penjelasan lebih?
6. Bolehkah saya menyalin kode untuk pengaplikasian di web saya?
7. Bolehkah saya meminta jaringan sosial media untuk komunikasi pribadi?

Sekian,
Terima kasih

**Comment by lintangtimur** [comment, marked as answer]
1. Full aplikasi OVO
5. Install dengan composer, penggunaan sudah dijelaskan di README.md
6. Boleh, MIT license.

**Reply by el-ang** [reply]
Thank you, kak

**Comment by riagie** [comment]
untuk mengetahui url yang di gunakan pada aplikasi ovo menggunakan aplikasi apa ya? untuk capture nya

**Reply by ii64** [reply]
Burp Suite, Charles Proxy, etc etc

**Reply by ii64** [reply]
Please open new discussion as this one is already answered, thank you
